Hey guys, after careful considering my updated code, please approve it.

paveldanek opened the pull request.

paveldanek committed:
This is a test.

paveldanek committed:
Merge remote-tracking branch 'upstream/main' into main

paveldanek committed:
Merge remote-tracking branch 'upstream/test' into main

paveldanek committed:
Merge remote-tracking branch 'upstream/main' into main

paveldanek committed:
Updates on classes 2/24.

Not sure about the generic-ness of it... Maybe needs to be re-worked...

paveldanek committed:
Added Serializable; Facade and UI now ready for coding.

paveldanek committed:
Cleaned up. Contains methods for an easy user input.

paveldanek committed:
Added Serializable; Facade and UI now ready for coding.

carterclark commented on src/store/entities/Product.java:
```diff
@@ -1,21 +1,24 @@
 		this.currentPrice = currentPrice;
 		this.stockOnHand = stockOnHand;
 		this.reOrderLevel = reOrderLevel;
-		this.id = "P-" + idCounter++;
```
Could we keep this in? I thought it made sense to have member id's be M-# and product id's be P-#

carterclark replied:
Would like this to be in

carterclark commented on src/store/entities/Product.java:
```diff
@@ -1,21 +1,24 @@
 		this.currentPrice = currentPrice;
 		this.stockOnHand = stockOnHand;
 		this.reOrderLevel = reOrderLevel;
-		this.id = "P-" + idCounter++;
```
Could we keep this in? I thought it made sense to have member id's be M-# and product id's be P-#

paveldanek replied:
It does make sense but Brahma wants--in the case of Product--the product number be manually input...

carterclark replied:
I see

paveldanek committed:
Trying to re-commit.

paveldanek committed:
Turns out Checkout class may not be necessary. We'll use Transaction instead.

paveldanek committed:
A small update on the UI (today's date generator).

paveldanek committed:
Fixed error catch block.

paveldanek committed:
UI almost ready for coding functionality.

We still need to include the data transfer pattern, as required by the assignment.

paveldanek committed:
I hope I resolved the conflicts.

paveldanek committed:
I hope I resolved the conflicts.

paveldanek committed:
Update Product.java
